creator-node/test/nodesync.test.js

```diff
@@ -643,6 +644,9 @@ describe('test nodesync', async function () {
     })
 
     it('Syncs correctly from clean user state with mocked export object', async function () {
+      // Added to address non-determinism in tests
+      await utils.timeout(1000)
```

Comment:
is the change here you pulled stuff out of the setTimeout callback?

Reply:
yep exactly - prob need to look at this more closely but it started failing for me without this change, idk why

Reply:
also functionally is same right - except for the added timeout at end of function

Reply:
yeah just confirming those two timeout calls are the only changes, as far as i could tell that was it

Reply:
yep that was it, but i also ended up reverting that since it didn't work - will address separately